src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -151,7 +151,7 @@ Here are some options to fix this error:
     <value>ASP.NET Core
 ------------
 Successfully installed the ASP.NET Core HTTPS Development Certificate.
-To trust the certificate (Windows and macOS only) first install the dev-certs tool by running 'dotnet tool install dotnet-dev-certs -g --version 2.1.0-preview1-final' and then run 'dotnet-dev-certs https --trust'.
+To trust the certificate run 'dotnet-dev-certs https --trust' (Windows and macOS only).
```

Comment:
Shouldn't this say `dotnet dev-certs`. dotnet-dev-certs is not being installed by the SDK as global tools, but rather as a native verb of the CLI.